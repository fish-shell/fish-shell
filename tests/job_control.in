# vim: ft=fish tw=80 :

# Regression test for #4178
set -l test (mktemp)
echo 'function confirm
	read -l answer < /dev/tty
	echo "answer=|$answer|"
end

confirm' > $test

set -l fish (status fish-path)
echo yes | command $fish $test
rm -f $test
