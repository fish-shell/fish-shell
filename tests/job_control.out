[30m(B[m]0;fish /mnt/d/rand/fish[?2004h[30m(B[m[2mÂ¶(B[m                                                                               Â¶ [K]0;fish /mnt/d/rand/fish[32mread[30m(B[m> [Kyes
[30m(B[m[30m(B[manswer=|yes|
[?2004l
